### The Lottery Problem 

$$ E = p_1v_1 + p_2v_2 + p_3v_3 $$ 

$$ p_1 = \frac{1}{100} * \frac{99}{100} * \frac{99}{100} $$ 
$$ p_2 = \frac{1}{100} * \frac{1}{100} * \frac{99}{100} $$ 
$$ p_3 = \frac{1}{100} * \frac{1}{100} * \frac{1}{100} $$ 



$$ v_1 = 200 $$ 

$$ v_2 = 5000 $$ 

$$ v_3 = 100,000 $$ 

In [1]:
p1 = (1/100) * (99/100) * (99/100)
p2 = (1/100) * (1/100) * (99/100)
p3 = (1/100) * (1/100) * (1/100)

v1 = 200
v2 = 5000
v3 = 100000

E = p1*v1 + p2*v2 + p3*v3 

print(E)

2.5552


$$ E \approx 2.5552 $$

On average, you can expect to gain 2.55 dollars. Due to this, the game is technically not fair, but it is in your favor. 